pages/getting_started/Getting_started.md

```diff
@@ -0,0 +1,35 @@
+## Getting Started
+
+**Depositing on dYdX Chain**
+
+1. Connect your preferred Ethereum wallet to the dYdX Chain deployment of your choice (i.e. [dydx.trade](https://dydx.trade)). _Support for Cosmos native wallets such as Keplr and Leap coming soon_.
+2. Deposit USDC to your DYDX chain address. The default onboarding path uses Circle's Cross Chain Transfer Protocol (CCTP) on Noble Chain. You can deposit USDC from many origination chains. Read more [here](https://dydx.exchange/blog/cctp).
+
+## User Journeys 
+
+**Trading via API**
```

Comment:
Nit: can you make the ordering Web -> Mobile -> API? Retail traders are least likely to dig through documentation imo